windows.md

```diff
@@ -40,19 +40,19 @@ With those compatibility things out of the way, you're ready to start the system
 5. Close PowerShell and open it again as administrator (like in step 2)<br><br>
 6. Copy the following text and right-click in the blue middle part of the PowerShell window to paste the text. Hit enter.<br><br>
    ```bash
-   choco install git nodejs-lts vscode hyper httpie flyctl -y
+   choco install git nodejs-lts vscode hyper httpie flyctl -y -f
```

Comment:
Let's switch to full flags `--yes --force`

https://docs.chocolatey.org/en-us/choco/commands/#default-options-and-switches

Reply:
ac2b6b4326d4cf5f5d372d1b920fe2c0e902f263